care/facility/migrations/0405_bed_name_constraint.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.2.8 on 2023-12-26 06:17
+
+from django.db import migrations, models
+
+
+def copy_bed_name(apps, schema_editor):
+    Bed = apps.get_model("facility", "Bed")
+    Bed.objects.update(old_name=models.F("name"))
+
+
+def reverse_copy_bed_name(apps, schema_editor):
+    Bed = apps.get_model("facility", "Bed")
+    Bed.objects.filter(old_name__isnull=False).update(name=models.F("old_name"))
+
+
+def fix_duplicate_bed_names(apps, schema_editor):
+    Bed = apps.get_model("facility", "Bed")
+    duplicates = (
+        Bed.objects.values("location", "name")
+        .annotate(count=models.Count("pk"))
+        .filter(count__gt=1)
+    )
```

Comment:
This does not account for case-insensitive duplicates right?

Reply:
No, currently not